hca_ingest/downloader/schema_url.py

```diff
@@ -0,0 +1,14 @@
+from dataclasses import dataclass, field
+
+
+@dataclass(unsafe_hash=True)
+class SchemaUrl:
```

Comment:
Have you considered using [urllib.parse](https://docs.python.org/3/library/urllib.parse.html#module-urllib.parse) for this?

Reply:
I have literally ported the old code to the new implementation, I could experiment with how to get the same results out of url lib.parse, but it seems out of scope.